config/jwt.txt

```diff
@@ -0,0 +1 @@
+0x5f7af7b44f9ca407319c2a133c26f782c89ec84e8112e19c5ca83f2c21a754b8
```

Comment:
How is this address generated?

Reply:
It's an auto-generated JWT secret required by OP Stack. I just removed it since I realized that running the Docker file would actually update this secret each time.